kolibri/content/migrations/0001_initial.py

```diff
@@ -101,7 +100,7 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='FileFormat',
             fields=[
-                ('extension', models.CharField(max_length=40, primary_key=True, serialize=False)),
+                ('extension', models.CharField(choices=[(b'.avi', 'avi'), (b'.mp4', 'mp4'), (b'.mov', 'mov'), (b'.webm', 'webm'), (b'.flv', 'flv'), (b'.mkv', 'mkv'), (b'.rm', 'rm'), (b'.rmvb', 'rmvb'), (b'.ssf', 'ssf'), (b'.mp3', 'mp3'), (b'.aac', 'aac'), (b'.ogg', 'ogg'), (b'.html', 'html'), (b'.css', 'css'), (b'.pdf', 'pdf'), (b'.txt', 'txt'), (b'.png', 'png'), (b'.jpeg', 'jpeg'), (b'.bmp', 'bmp'), (b'.gif', 'gif'), (b'.svg', 'svg')], max_length=40, primary_key=True, serialize=False)),
```

Comment:
But don't think we need choices for this. Nobody will ever be selecting them from a list.